httpd-static/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine AS release
+RUN apk --quiet --no-cache --no-progress add curl \
```

Comment:
Here is my version without `curl`:
```Dockerfile
FROM alpine:3.8 AS release
ARG SERVE_VERSION="0.3.0"
WORKDIR /srv
ADD https://github.com/syntaqx/serve/releases/download/v${SERVE_VERSION}/serve_${SERVE_VERSION}_linux_x86_64.tar.gz /tmp
RUN tar -zxvf /tmp/serve_${SERVE_VERSION}_linux_x86_64.tar.gz serve -C /tmp \
  && cp /tmp/serve /bin/serve \
  && chmod +x /bin/serve && rm -rf /tmp/*
ENTRYPOINT ["serve"]
EXPOSE 8080
```
It's 11.5 MB instead of 18.1 MB and uses the default `serve` port `8080`. Furthermore I use specific versions for `alpine` and `serve`. WDYT?

Reply:
Good idea. No dependencies. But I think more robust is this:
```
RUN wget -P /tmp $(wget -q -O - https://api.github.com/repos/syntaqx/serve/releases/latest | grep 'browser_download_url.*linux_x86_64' | awk '{ print $2 }' | tr -d '"') \
    && ...
```
`wget` is part of Alpine. Also we can't forget to update. To further improve this we could get the platform via `echo "$(uname -o | tr '[:upper:]' '[:lower:]')_$(uname -m)"`.

Reply:
As long as we use the same platform inside the docker image (`linux_x86_64`) we could use a static string for this.

Reply:
Agree, static string it is ;) If the rest is ok with you I would update the PR.

Reply:
Do it 👍

Reply:
Comitted, please verify